Makefile

```diff
@@ -1,125 +1,152 @@
-# Project Environment Variables
+############################################################
+# Makefile for the Anchore CLI, a simple command line interface to the
+# Anchore Engine service. The rules, directives, and variables in this
+# Makefile enable testing, Docker image generation, and pushing Docker
+# images.
+############################################################
+
+
+#### Docker Hub, git repos
+############################################################
 DEV_IMAGE_REPO := anchore/anchore-cli-dev
+PROD_IMAGE_REPO := anchore/anchore-cli
+TEST_HARNESS_REPO := https://github.com/anchore/test-infra.git
 
-# Environment variables set in CircleCI environment
+
+#### CircleCI environment variables
+############################################################
 export VERBOSE ?= false
 export CI ?= false
-# DOCKER_USER & DOCKER_PASS are declared in CircleCI contexts
 export DOCKER_PASS ?=
 export DOCKER_USER ?=
-# LATEST_RELEASE_BRANCH is declared in the CircleCI project environment variables settings
 export LATEST_RELEASE_BRANCH ?=
 export PROD_IMAGE_REPO ?=
 export RELEASE_BRANCHES ?=
 
+# without the following line, the line below
+# (ACTIVATE_VENV := source $(VENV)/bin/activate)
+# fails, oddly. when i change source to . it works
+# even if SHELL := ... is commented out
+#SHELL := /usr/bin/env bash
+
 # Use $CIRCLE_BRANCH if it's set, otherwise use current HEAD branch
 GIT_BRANCH := $(shell echo $${CIRCLE_BRANCH:=$$(git rev-parse --abbrev-ref HEAD)})
+
 # Use $CIRCLE_PROJECT_REPONAME if it's set, otherwise the git project top level dir name
 GIT_REPO := $(shell echo $${CIRCLE_PROJECT_REPONAME:=$$(basename `git rev-parse --show-toplevel`)})
 TEST_IMAGE_NAME := $(GIT_REPO):dev
+
 # Use $CIRCLE_SHA if it's set, otherwise use SHA from HEAD
 COMMIT_SHA := $(shell echo $${CIRCLE_SHA:=$$(git rev-parse HEAD)})
+
 # Use $CIRCLE_TAG if it's set
 GIT_TAG ?= $(shell echo $${CIRCLE_TAG:=null})
 
-# Make environment configuration
+CLUSTER_NAME := e2e-testing
+
+
+# Environment configuration for make
+############################################################
 VENV := venv
+ACTIVATE_VENV := . $(VENV)/bin/activate
 PYTHON := $(VENV)/bin/python3
-.DEFAULT_GOAL := help # Running `Make` will run the help target
-.NOTPARALLEL: # wait for targets to finish
+CI_USER := circleci
```

Comment:
Don't think we need this anymore since it's no longer using docker

Reply:
Great, I'll remove it.